### PR TITLE
Removed isHoustonDeployed checks.

### PR DESCRIPTION
**Pull request by 0xfornax**

### CODE REVIEW

File: rocketpool/api/node/status.go [hunk 1]
```diff
@@ -93,7 +86,7 @@ func getStatus(c *cli.Context) (*api.NodeStatusResponse, error) {
 
 	// Get node details
 	wg.Go(func() error {
-		details, err := node.GetNodeDetails(rp, nodeAccount.Address, response.IsHoustonDeployed, nil)
+		details, err := node.GetNodeDetails(rp, nodeAccount.Address, true, nil)
```
Comment (jshufro) [inline rocketpool/api/node/status.go:89]: I'd prefer to see this argument deleted in rocketpool-go first, and then this PR should have a single commit that updates go.mod/sum and removes calls that include this arg

**Comment by jshufro** [comment]
Odd- I can't find v1.8.3 of rocketpool-go (referenced in go.mod) https://github.com/rocket-pool/rocketpool-go/tags

File: rocketpool/api/node/status.go (hunk 1 above)
Comment (0xfornax) [reply rocketpool/api/node/status.go:89]: https://github.com/rocket-pool/rocketpool-go/pull/26

**Comment by 0xfornax** [comment]
> Odd- I can't find v1.8.3 of rocketpool-go (referenced in go.mod) https://github.com/rocket-pool/rocketpool-go/tags

I've pushed the v1.8.3 tag.

File: rocketpool/api/network/stats.go [hunk 2]
```diff
@@ -128,7 +128,7 @@ func getStats(c *cli.Context) (*api.NetworkStatsResponse, error) {
 	wg.Go(func() error {
 		multicallerAddress := common.HexToAddress(cfg.Smartnode.GetMulticallAddress())
 		balanceBatcherAddress := common.HexToAddress(cfg.Smartnode.GetBalanceBatcherAddress())
-		contracts, err := rpstate.NewNetworkContracts(rp, multicallerAddress, balanceBatcherAddress, false, nil)
```
Comment (jshufro) [inline rocketpool/api/network/stats.go:131]: cool- great example of why it's good to update the library to remove the argument first. we missed this call earlier.

**Review by jshufro — APPROVED** [review]
If we wanted to be _really_ pedantic about this, I would insist on using `git rebase` to reorder the commits so the rocketpool-go dependency is updated before the other changes, but meh, up to you.

**Pull request merged**